interfaces/Portalicious/src/app/pages/project-payments/components/payment-summary-card/payment-summary-card.component.ts

```diff
@@ -0,0 +1,95 @@
+import { CommonModule, CurrencyPipe } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  inject,
+  input,
+} from '@angular/core';
+import { RouterLink } from '@angular/router';
+
+import { injectQuery } from '@tanstack/angular-query-experimental';
+import { CardModule } from 'primeng/card';
+import { SkeletonModule } from 'primeng/skeleton';
+
+import { AppRoutes } from '~/app.routes';
+import { ColoredChipComponent } from '~/components/colored-chip/colored-chip.component';
+import { MetricContainerComponent } from '~/components/metric-container/metric-container.component';
+import { SkeletonInlineComponent } from '~/components/skeleton-inline/skeleton-inline.component';
+import { PaymentApiService } from '~/domains/payment/payment.api.service';
+import { TranslatableStringPipe } from '~/pipes/translatable-string.pipe';
+
+@Component({
+  selector: 'app-payment-summary-card',
+  standalone: true,
+  imports: [
+    CardModule,
+    SkeletonModule,
+    TranslatableStringPipe,
+    RouterLink,
+    CommonModule,
+    CurrencyPipe,
+    MetricContainerComponent,
+    SkeletonInlineComponent,
+    ColoredChipComponent,
+  ],
+  templateUrl: './payment-summary-card.component.html',
+  styles: ``,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PaymentSummaryCardComponent {
+  private paymentApiService = inject(PaymentApiService);
+
+  public projectId = input.required<number>();
+  public paymentId = input.required<number>();
+  public paymentDate = input.required<string>();
+  public cardIndex = input.required<number>();
+
+  public metrics = injectQuery(() => ({
+    ...this.paymentApiService.getPayment(this.projectId, this.paymentId)(),
+    enabled: !!this.projectId() && !!this.paymentId(),
+  }));
```

Comment:
```suggestion
  public metrics = injectQuery(this.paymentApiService.getPayment(this.projectId, this.paymentId));
```

Again, I don't see how either of those can ever be false (because the inputs are `required`)